sdks/index.html

```diff
@@ -53,7 +53,7 @@
       </thead>
       <tr>
         <td class="text-right">
-          <a href="https://docs.aerogear.org/aerogear/latest/mobile-developer-console.html#android">Android
+          <a>Android</a>
```

Comment:
```suggestion
          Android
```
do we need the a tag?

Reply:
It's been a while since I've worked with HTML :) ... but after a quick check, it seems that the <a> tag is redundant in the example above :+1: